python/mxnet/base.py

```diff
@@ -47,6 +48,32 @@
     integer_types = (int, long, np.int32, np.int64)
     py_str = lambda x: x
 
+
+def data_dir_default():
+    """
+
+    :return: default data directory depending on the platform and environment variables
+    """
+    system = platform.system()
+    if system == 'Windows':
+        return os.environ.get('APPDATA')
+    elif system == 'Linux' or system == 'Darwin':
+        if 'XDG_DATA_HOME'in os.environ:
```

Comment:
what's XDG? why?

Reply:
freedesktop convention https://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html

Reply:
Have we ever followed this spec anywhere? It doesn't seem necessary.